fsspec/tests/conftest.py

```diff
@@ -17,6 +17,186 @@
 ).read()
 win = os.name == "nt"
 
+GLOB_TESTS = [
```

Comment:
Can you add a comment here explaining what the 4 tuple elements are?

Reply:
I added the "argnames" directly inside this object alongside the `argvalues" (both argument names of the `pytest.mark.parameterize` method) instead so it is clearer what the elements are.